src/EnergyPlus/CondenserLoopTowers.cc

```diff
@@ -349,24 +349,24 @@ namespace CondenserLoopTowers {
                 tower.DesInletAirDBTemp = 35.0;
                 tower.TowerInletCondsAutoSize = true;
             }
-            tower.DesInletAirWBTemp = NumArray(14);
-            if (tower.DesInletAirWBTemp == 0) {
-                tower.DesInletAirWBTemp = 25.6;
+            tower.DesignInletWB = NumArray(14);
+            if (tower.DesignInletWB == 0) {
+                tower.DesignInletWB = 25.6;
                 tower.TowerInletCondsAutoSize = true;
             }
-            tower.DesApproach = NumArray(15);
-            if (tower.DesApproach == DataSizing::AutoSize || tower.DesApproach == 0) {
-                tower.DesApproach = 3.9;
+            tower.DesignApproach = NumArray(15);
```

Comment:
This is CoolingTower:SingleSpeed, Range and Approach where stored in the wrong variable as weel

https://github.com/NREL/EnergyPlus/blob/31e3c33467c5873371bf48b12a7318215971c315/src/EnergyPlus/CondenserLoopTowers.cc#L3375-L3376

Take 5ZoneBoilerOutsideAirReset.idf, which has a CT with defaulted Range and Approach.

Develop:

![image](https://github.com/user-attachments/assets/2f035fd4-22e1-4c61-8799-411491997b0f)

This PR:

![image](https://github.com/user-attachments/assets/c616a6b1-c7b4-46ac-ba93-3680a102fe0c)